fc/language/statements.py

```diff
@@ -53,7 +23,7 @@ def __init__(self, names, rhs, optional=False):
     def Evaluate(self, env):
         try:
             results = self.rhs.Evaluate(env)
-        except:
+        except BaseException:
```

Comment:
Why are we catching system-exciting exceptions here? (Rather than catching Exception)

Reply:
Because that's what 2to3 did by default and I didn't think about it. I'll change these.